src/components/HomePage/SiteMask.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import styles from './styles/SiteMask.css';
+
+let cx = classNames.bind(styles);
+
+class SiteMask extends React.Component {
```

Comment:
When reading through the React documentation I remember reading that you should make each component deal with one thing. I thought making a separate component for the masking of the site while the Gifs loaded in the VideoBackground div would have been best but looking how things are going now this PR might be a bit stale.